fdbrpc/FlowGrpcTests.cpp

```diff
@@ -31,6 +34,21 @@ void forceLinkGrpcTests2() {}
 namespace fdbrpc_test {
 namespace asio = boost::asio;
 
+std::string generate_random_string(int size) {
```

Comment:
There is a `RandomStringGenerator` you can use.

Reply:
I had to use deterministicRandom() because RandomStringGenerator is in fdbclient module, and this just depends on fdbrpc. Can move that utility out of that module in later PR.